data/proposals.js

```diff
@@ -468,5 +468,45 @@ module.exports = [
       "gstepniewski"
     ],
     "wiki": "Wilfred (U.S. TV series)"
+  }, 
+  {
+    "title": "Daredevil",
```

Comment:
Hey @ryanburgess I have this one [on the 'already watched' list](https://github.com/michalbe/tv-series/blob/master/data/data.js#L153-L159), pls remove and update the PR.